components/LayoutWithAuth0.js

```diff
@@ -91,8 +112,9 @@ class LayoutWithAuth0 extends Component {
       // TODO: find a better way to pass props to children.
       const extraProps = {
         ...this.state,
-        createUser: this.createUser,
-        handleSignup
+        createApiUser: this.createApiUser,
+        handleSignup,
+        updateUser: this.updateUser
```

Comment:
I'm kind of stuck on this. It sure looks like there is potential for this list of handlers to grow and grow and grow...

Reply:
I'm definitely open to ideas.